config/common/operator/deployment-operator.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         - name: dynatrace-operator
           args:
             - operator
-          image: quay.io/dynatrace/dynatrace-operator:snapshot
+          image: quay.io/dynatrace/dynatrace-operator:snapshot-validating-webhook
```

Comment:
I guess this has been added by accident - please remove

Reply:
removed